src/probabilities_estimators/histograms/rectangular_binning.jl

```diff
@@ -15,35 +18,113 @@ Binning instructions are deduced from the type of `ϵ` as follows:
     intervals that cover all data.
 4. `ϵ::Vector{Float64}` divides the i-th coordinate axis into intervals of fixed size
     `ϵ[i]`, starting from the axis minima until the data is completely covered by boxes.
+
+!!! note "Be aware when mixing methods"
+    If the range of the data along some dimension is `[r1, r2]`, then
+    `RectangularBinning(N)` results in a bin edge length of
+    `nextfloat(r2 - r1) / N)`. `nextfloat` is used to ensure data are completely
+    covered. Keep this in mind if you're switching between the `ϵ::Int` and `ϵ::Float64`
+    ways of constructing grids.
 """
 struct RectangularBinning{E} <: AbstractBinning
     ϵ::E
 end
 
+const ValidFixedBinInputs = Union{Number, NTuple}
+
 """
-    RectangularBinMapping <: Encoding
+    FixedRectangularBinning <: AbstractBinning
```

Comment:
```suggestion
    FixedRectangularBinning <: Encoding
```
This is the information that is actually useful for a user.